components/provisioner/assets/database/provisioner.sql

```diff
@@ -35,6 +35,8 @@ CREATE TABLE gardener_config
     auto_scaler_max integer NOT NULL,
     max_surge integer NOT NULL,
     max_unavailable integer NOT NULL,
+    auto_update_kubernetes_version boolean NOT NULL,
```

Comment:
I wonder if the name is not a little misleading. What do you think about `enable_kubernetes_auto_update`? The same is the case for other files.

Reply:
Yeah, your suggestion is totally right. I used the names of the Gardener structure fields but in this case, this will be better